changelogs/fragments/10346-jenkins-plugins-fixes.yml

```diff
@@ -0,0 +1,6 @@
+bugfixes:
+  - "jenkins plugins plugin - install latest compatible version instead of latest (https://github.com/ansible-collections/community.general/pull/10346)."
+  - "jenkins plugins plugin - seperate Jenkins and external url credentials (https://github.com/ansible-collections/community.general/pull/10346)."
+
+minor_changes:
+  - "jenkins plugins plugin - install dependencies for specific version (https://github.com/ansible-collections/community.general/pull/10346)."
```

Comment:
```suggestion
  - "jenkins_plugin - install dependencies for specific version (https://github.com/ansible-collections/community.general/issues/4995, https://github.com/ansible-collections/community.general/pull/10346)."
```

Reply:
This is still pending - there should be two URLs listed (one for the bug, one for this PR), as in Felix's suggestion. Same for the other lines above/below.